3 - PB/Documentazione interna/Verbali/24-03-17/tasks.json

```diff
@@ -0,0 +1,247 @@
+[
+	{
+	  "Descrizione": "Scrivere guida per Impostazioni",
+	  "ID task": "DOC-546",
+	  "Ruolo": "Programmatore"
+	},
+	{
+	  "Descrizione": "Scrivere guida per Modifica zona",
+	  "ID task": "DOC-559",
+	  "Ruolo": "Programmatore"
+	},
+	{
+	  "Descrizione": "Scrivere guida per Creazione zona",
+	  "ID task": "DOC-560",
+	  "Ruolo": "Programmatore"
+	},
+	{
+	  "Descrizione": "Scrivere guida per Configurazione ambiente",
+	  "ID task": "DOC-563",
+	  "Ruolo": "Programmatore"
+	},
+	{
+	  "Descrizione": "Aggiornare dashboard di monitoraggio",
+	  "ID task": "DOC-585",
+	  "Ruolo": "Amministratore"
+	},
+	{
+	  "Descrizione": "Redigere preventivo di periodo Sprint 20",
+	  "ID task": "DOC-586",
+	  "Ruolo": "Amministratore"
+	},
+	{
+	  "Descrizione": "Redigere consuntivo di periodo Sprint 19",
+	  "ID task": "DOC-590",
+	  "Ruolo": "Amministratore"
+	},
+	{
+	  "Descrizione": "Aggiornare descrizione repo WMS3",
+	  "ID task": "DOC-602",
+	  "Ruolo": "Amministratore"
+	},
+	{
+	  "Descrizione": "Revisionare sezione Transition process",
+	  "ID task": "DOC-605",
+	  "Ruolo": "Responsabile"
+	},
+	{
+	  "Descrizione": "Revisionare sezione Validation process",
+	  "ID task": "DOC-607",
+	  "Ruolo": "Responsabile"
+	},
+	{
+	  "Descrizione": "Revisionare sezione Verification process",
+	  "ID task": "DOC-608",
+	  "Ruolo": "Responsabile"
+	},
+	{
+	  "Descrizione": "Revisionare sezione Integration process",
+	  "ID task": "DOC-609",
+	  "Ruolo": "Responsabile"
+	},
+	{
+	  "Descrizione": "Revisionare sezione Implementation process",
+	  "ID task": "DOC-610",
+	  "Ruolo": "Responsabile"
+	},
+	{
+	  "Descrizione": "Revisionare sezione Quality management process",
+	  "ID task": "DOC-611",
+	  "Ruolo": "Responsabile"
+	},
+	{
+	  "Descrizione": "Confermare incontro settimanale",
+	  "ID task": "DOC-612",
+	  "Ruolo": "Responsabile"
+	},
+	{
+	  "Descrizione": "Caricare verbali firmati nel repository",
+	  "ID task": "DOC-613",
+	  "Ruolo": "Verificatore"
+	},
+	{
+	  "Descrizione": "Inviare email al Referente aziendale con verbali da firmare",
+	  "ID task": "DOC-614",
+	  "Ruolo": "Responsabile"
+	},
+	{
+	  "Descrizione": "Inviare email al Prof. Cardin per sottoporre documento aggiornato",
+	  "ID task": "DOC-615",
+	  "Ruolo": "Responsabile"
+	},
+	{
+	  "Descrizione": "Inviare email al Prof. Cardin per individuazione diagramma dei componenti",
+	  "ID task": "DOC-616",
+	  "Ruolo": "Responsabile"
+	},
+	{
+	  "Descrizione": "Redigere paragrafo sui componenti",
+	  "ID task": "DOC-617",
```

Comment:
Di questa cosa mi sono accorto ieri facendo il preventivo: DOC-617, DOC-618 e DOC-619 vanno tolte perché erano un duplicato di una sottotask della Specifica Tecnica, e il capitolo l'avevi già scritto durante lo Sprint del verbale, infatti giustamente l'hai segnato come lavoro svolto. C'era stata un po' di confusione in pianificazione